src/extensions/default/analyzers/Microsoft.DotNet.UpgradeAssistant.Extensions.Default.Analyzers/Microsoft.DotNet.UpgradeAssistant.Extensions.Default.Analyzers.csproj

```diff
@@ -3,7 +3,7 @@
     <TargetFramework>netstandard2.0</TargetFramework>
     <IsPackable>false</IsPackable>
     <!-- Avoid ID conflicts with the package project. -->
-    <PackageId>*$(MSBuildProjectFullPath)*</PackageId>
+    <PackageId>Real.Microsoft.DotNet.UpgradeAssistant.Extensions.Default.Analyzers</PackageId>
```

Comment:
Why `Real.`?

Reply:
Oh, it's this nightmare with PackageID here too. Ugh, if something gets packaged as a NuGet then there will be a broken NuGet dependency for this. I have no idea how to fix that.

Reply:
@mgoertz-msft 

The Microsoft.DotNet.UpgradeAssistant.Extensions.Default.Analyzers.Package.csproj file uses the PackageId Microsoft.DotNet.UpgradeAssistant.Extensions.Default.Analyzers, so the Microsoft.DotNet.UpgradeAssistant.Extensions.Default.Analyzers.csproj can't use the same PackageId and has to use something unique (hence the previous use of the FullPath of the .csproj).

Since the .Package project is what is responsible for packaging this project's build output (along with some other projects that are dependencies), there is no need for this project to have a PackageId that would be usable for publishing, it just needs to be unique within the solution (afaik).